tools/offwaketime.py

```diff
@@ -20,6 +20,16 @@
 
 # arg validation
 def positive_int(val):
+    dest = []
```

Comment:
The change of this function from returning a int to a list caused the following error
```
  File "../../tools/offwaketime.py", line 125, in <module>
30:     duration = int(args.duration)
30: TypeError: int() argument must be a string or a number, not 'list'
```

Reply:
Yes the fix is to use _positive_int for the type of duration. I did catch it and updated my branch. I see that the commit has this updated.

Reply:
> Yes the fix is to use _positive_int for the type of duration. I did catch it and updated my branch. I see that the commit has this updated.

The error still there. Could you double check?

Reply:
Ops, I had missed the usage of positive_int in positive_nonzero_int. Pushed fixed branch now. Restart the tests.